browser/init.js

```diff
@@ -1,7 +1,6 @@
 var jQuery = require("funcunit/browser/jquery");
 
 var FuncUnit = window.FuncUnit || {};
```

Comment:
What I mean is, if `global.js`'s purpose is to avoid setting globals, then we shouldn't set the FuncUnit global in browser/init.js.  We should set it in global.js as well.